connectors/byoc.py

```diff
@@ -630,11 +638,16 @@ async def sync(self, elastic_server, idling, sync_now=False):
 
         try:
             service_type = self.service_type
-            if not sync_now:
+            if sync_now:
+                self.sync_now = True
+                logger.info("Sync forced")
```

Comment:
should it be `debug`?

Reply:
I haven't changed this log level, but to me it makes sense to keep it in info - it's good to have indication why sync was started even in INFO IMO, what do you think?